backend/src/main/resources/db/migration/V1.18__add_compression_dict_table.sql

```diff
@@ -0,0 +1,8 @@
+create table compression_dictionaries
+(
+    id            SERIAL primary key,
+    hash          TEXT not null unique,
```

Comment:
```suggestion
    hash          char(64) not null unique,
```
We know it's 64 chars as we use sha-256